src/logger/src/metrics.rs

```diff
@@ -489,6 +499,33 @@ pub struct SerialDeviceMetrics {
     pub write_count: SharedMetric,
 }
 
+/// Metrics related to signals.
+#[derive(Default, Serialize)]
+pub struct SignalMetrics {
+    /// Number of times that SIGBUS was handled.
+    pub sigbus: SharedMetric,
+    /// Number of times that SIGSEGV was handled.
+    pub sigsegv: SharedMetric,
+}
+
+#[cfg(target_arch = "x86_64")]
+/// Metrics related to snapshot.
+// These store the duration of creating/loading a snapshot.
+// If there are more than one `/snapshot/create` request in a minute
+// (until the metrics are flushed), only the duration of the last
+// snapshot creation is stored in the metric. If the user is interested
+// in all the durations, a `FlushMetrics` request should be sent after
+// each `create` request.
+#[derive(Default, Serialize)]
+pub struct SnapshotMetrics {
+    /// Measures the snapshot full create time in microseconds.
+    pub last_full_create_us: SharedMetric,
```

Comment:
nit: I think `last_` is too verbose and doesn't add value, metrics usually express the last instance of some event

Reply:
I was thinking that this way it will be clearer that we don't store all the `create` values, but since this is the default behavior, it makes sense to not add `last_`.